ecr/templates/auth_server_policy.json.tpl

```diff
@@ -0,0 +1,20 @@
+{
+  "Version": "2008-10-17",
+  "Statement": [
+    {
+      "Sid": "AllowPullConsignmentAPI",
+      "Effect": "Allow",
+      "Principal": {
+        "AWS": [
+          "arn:aws:iam::${intg_account}:role/keycloak_ecs_execution_role_intg",
```

Comment:
Do we need the production account here aswell?

Reply:
We can't. If the role doesn't exist you get an error. For most things you can get away with non-existent role it seems but in this case, AWS checks.

Reply:
Fair enough.